src/datatable/datatable.jl

```diff
@@ -758,6 +758,30 @@ function nullable!{T <: ColumnIndex}(dt::DataTable, cols::Vector{T})
     end
     dt
 end
+function nullable!(dt::DataTable)
```

Comment:
Even if that only saves a few lines, this can be replaced with `cols::Vector{T}=names(dt)` in the previous definition.